src/c/user-billing.js

```diff
@@ -50,12 +49,12 @@ const userBilling = {
                 name: 'Banco Bradesco S.A.'
             }];
 
-        userVM.getUserBankAccount(user_id).then(data => bankAccount(_.first(data))).catch(err => {
+        userVM.getUserBankAccount(userId).then(data => bankAccount(_.first(data))).catch(err => {
```

Comment:
I think we could define a function for those error handlers, just to avoid repeating them. Then just do `.catch(handleError)`.